rules/manager.go

```diff
@@ -330,17 +330,19 @@ func (g *Group) run(ctx context.Context) {
 			return
 		}
 		go func(now time.Time) {
+			for _, rule := range g.seriesInPreviousEval {
+				for _, r := range rule {
```

Comment:
You can do `g.staleSeries = append(g.staleSeries, rule...)`

Reply:
cannot use rules (type map[string]labels.Labels) as type labels.Labels in append

Reply:
I will improve it

Reply:
Well not sure how to improve that